docs/jsonapi-spec.md

```diff
@@ -591,6 +592,83 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
 
 If an error occurs the command will not return `data`.
 
+### findOneAndReplace Command
+
+`findOneAndReplace` replaces a single document based on the specified filter clause. 
+
+*Syntax:*
+
+```bnf
+<find-one-and-replace-command> ::= findOneAndReplace <find-one-and-replace-command-payload> 
+<find-one-and-replace-command-payload> ::= <filter-clause>? 
+                             <sort-clause>? 
+                             <replacement-clause>?
+                             <find-one-and-replace-command-options>?
+
+<find-one-and-replace-command-options> ::= (<find-option-name>)*
```

Comment:
Probably needs value as well; I see `find-command` uses:

```
<find-command-options> ::= (<find-option-name> <find-option-value>,)*
```

Not sure if regular `find-option-name`/`-value` may be used since different Commands have different options.

Reply:
Find options won't fit here. This has only one option `returnDocument` which can have value before/after